app/views/articles/new.html.erb

```diff
@@ -1,54 +1,54 @@
-                    <h4>
-                        <%= pluralize(@article.errors.count, "error") %> prohibited this article from being saved:
-                    </h4>
-                    <% @article.errors.full_messages.each do |msg| %>
-                        <p class="form-errors">
-                            <%= msg %>
-                        </p>
-                    <% end %>
-                </div>
+  <div class="wrapper">
+    <h1>Create New Post:</h1>
+    <div class="form-element">
+      <%= form_for @article, url: {action: 'create'}, html: {class: 'new-form', multipart: true} do |form| %>
+
+        <% if @article.errors.any? %>
+          <div class="form-error">
+            <h4>
+              <%= pluralize(@article.errors.count, "error") %> prohibited this article from being saved:
+            </h4>
+            <% @article.errors.full_messages.each do |msg| %>
+              <p class="form-errors">
+                <%= msg %>
+              </p>
             <% end %>
-            
-            <p class="form-p">
-                <%= form.label :title, 'Title:' %>
-                <%= form.text_field :title, class: 'input-field' %>
-            </p>
-            
-            <p class="form-p">
-                <%= form.label :body, 'Content:' %>
-                <%= form.text_area :body, class: 'textarea-field' %>
-                <!-- 
-                    @source https://cdn.ckeditor.com/#ckeditor5
-               -->
-                <script>
-                  ClassicEditor
-                    .create( document.querySelector( '#article_body' ), {
-                        toolbar: [ 'heading', '|', 'bold', 'italic', 'bulletedList', 'numberedList', 'link', 'blockQuote' ]
-                    }
-                    ) .catch( error => {
-                        console.log( error );
-                    } );
-                </script> 
-            </p>
-            
-            <p class="form-p">
-                <%= form.label 'Upload picture:' %>
-                <%= form.file_field :content_img %>
-            </p>
-        
-            <p class="form-p">
-                <%= form.submit 'Create' %>
-            </p>
+          </div>
         <% end %>
-        </div>
+
+        <p class="form-p">
+          <%= form.label :title, 'Title:' %>
+          <%= form.text_field :title, class: 'input-field' %>
+        </p>
+
+        <p class="form-p">
+          <%= form.label :body, 'Content:' %>
+          <%= form.text_area :body, class: 'textarea-field' %>
+          <!-- 
+  @source https://cdn.ckeditor.com/#ckeditor5
+  -->
+          <script>
+            ClassicEditor
+              .create( document.querySelector( '#article_body' ), {
+              toolbar: [ 'heading', '|', 'bold', 'italic', 'bulletedList', 'numberedList', 'link', 'blockQuote' ]
+            }
+                     ) .catch( error => {
+              console.log( error );
+            } );
+          </script> 
```

Comment:
Inline javascript is generally frowned upon. What do you think about putting this into `application.js`?